usmqe/api/graphiteapi/graphiteapi.py

```diff
@@ -34,4 +35,7 @@ def get_datapoints(self, target, from_date=None, until_date=None):
         response = requests.get(
             pytest.config.getini("graphite_api_url") + pattern)
         self.check_response(response)
-        return response.json()[0]["datapoints"]
+        try:
+            return response.json()[0]["datapoints"]
+        except IndexError as err:
+            return response.json()
```

Comment:
I would rather use something like: `if len(response_json) > 0 and "datapoints" in response_json[0]:`